docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Sharing To & From GitHub Gists
+description: GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
+---
+
+# Import Gists from GitHub
+
+<Image zoom src="https://storage.googleapis.com/pieces_static_resources/pfd_wiki/IMPORT_GIST.gif" />
+
+Before proceeding to work with Gists from GitHub in Pieces, make sure that you are signed in. _If you are signed in to both Google and GitHub, be sure that the **primary emails on both accounts match**._
+
+## Sharing Pieces snippets as GitHub Gists
+
+GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
+
+<Image zoom alt={"Generating a shareable link in the Pieces desktop app"} src="https://storage.googleapis.com/pieces_static_resources/pfd_wiki/SHAREABLE_LINK.gif" />
+
+1. After signing in and ensuring that you are **connected to the cloud**, select any snippet in the desktop app.
+2. Click the "Shareable Link" icon on the Action menu nested on the right side of your screen.
+3. Select the GitHub logo to the right of your shareable link inside the input.
+    -If you have not authenticated with GitHub you will have to in order to generate a Gist.
+4. (Optional) Create a title and a description. You can create a title and a description from here, or you can skip this step and a generated title will be created for you.
+5. Press "Generate," and you'll get the link to the Gist, where it can be found on your list of public gists on your GitHub account.
+
+#### Bundled Metadata Preview
+Every Gist has a _Markdown Preview_ built using our <a target="_blank" href="{{ links.internal.context_awareness_engine.origin_details }}">Context Awareness Engine and Origin Details</a> which includes relevant and related information to the snippet's context.
+
+
+## How to Import Gists to Pieces Desktop
+
+
+To import the Gists and upgrade them:
+1. Navigate to 'Add Snippets' > 'Import Gists'
+2. The Snippet Discovery window will open, as we use our Snippet Discovery feature to [enrich your snippets](/features/auto-enrichment) as you import them from GitHub. This way, your Gists will maintain their context.
+3. Once you press "Select Gists," in a similar way as [Snippet Discovery](/product-highlights-and-benefits/in-project-snippet-discovery), you will be presented with a list of Snippets that have been discovered from your GitHub account.
+
+    If you're not seeing your Gists, make sure they are set to public. Otherwise, they will not be detected.
```

Comment:
im actually not sure if this is true, were you able to confirm or deny this ?

Reply:
From what I can tell yes this is accurate, what exactly are you suspicious of?

Reply:
just wanted to make sure that you have to have your gists set to public in order for pieces to discover them, if you can just try setting gists to private and then testing if you can discover them via Pieces that would be great